### DISCUSSION TITLE
Missing 1 XCH in cold storage address by checking chiaexplorer.com

### DISCUSSION THREAD

**Discussion by chenus1** [Chia-Network/chia-blockchain#8195] [Support]
Describe the bug

In the morning of 22 August 2021, I just checked the https://www.chiaexplorer.com/  for my cold storage address and find that 1 xch was spent and missing (see the snapshot of screen) and it is not show in the net balance. I did transfer this XCH on August 19, 2021 from my farmer address to this cold storage address.  My screen also shows that threat secured and we have successfully aborted connection tcp://125.68.143.131:8444
![image](https://user-images.githubusercontent.com/45975235/130358393-d6068c94-c985-4fe3-889b-2a614eb669ab.png)
![image](https://user-images.githubusercontent.com/45975235/130358464-7bc66e37-46c6-46eb-b828-0cd7eae6e194.png)

here is the snapshot of the coin 

![image](https://user-images.githubusercontent.com/45975235/130358580-a0088552-b935-4c88-9c39-1a68a242e9a3.png)
I traced that spend block index is 734446, which is linked to some pool address as below

![image](https://user-images.githubusercontent.com/45975235/130358701-13977319-73cf-41d4-bef0-b921f6864036.png)

Questions:

Is there any possible way to solve this problem / get my coins back?

Desktop

OS: Windows 8.1 Pro

CPU: AMD FX 8150 Ram 16GB

**Comment by loppefaaret** [comment]
`xch16dh6v6kg0t7d36gkkwmcxeqtaf77yagy7w828mugrkh8vjha8ujq8q4fqn`
First off, the fact that an outgoing transaction has taken place, this is _no longer a cold wallet_!
If you are 100% sure that you have not created a transaction from this wallet, *imidiately* transfer the balance out of the wallet!

**Comment by loppefaaret** [comment]
https://xchscan.com/address/xch16dh6v6kg0t7d36gkkwmcxeqtaf77yagy7w828mugrkh8vjha8ujq8q4fqn
This shows a initial 0.01 XCH going in 3~ months ago, and then activity in the last week...
are you absolutely sure that you have not by mistake imported this key on an online client, and started syncing the wallet ?
perhaps even mistakenly created the transaction itself ? it has taken the 1.00 XCH and split it into 0.01 & 0.989 as you can see on the above link
the 0.01 went to https://xchscan.com/address/xch1rg97f3dvk7wjlsjdcvrtxujv606swa42hpplfwlakuhqfaz0j4fs5x2p8y which is a very active address, and the 0.989 went to a unique address, which might be belonging to your "not so cold wallet any more" ?

**Comment by loppefaaret** [comment]
https://xchscan.com/address/xch1rg97f3dvk7wjlsjdcvrtxujv606swa42hpplfwlakuhqfaz0j4fs5x2p8y is also the same address that initially seeded the "not so cold wallet address" 3 months ago i see now. 
i guess you synced the "not so cold wallet" and sent back the 0.01 XCH from where it came ?

**Reply by chenus1** [reply]
Thank you for comments! yes. after I synced the "cold wallet" and checked to make sure that I have 1 XCH  balance in the cold wallet, then I sent 0.01 XCH to xch1rg97f3dvk7wjlsjdcvrtxujv606swa42hpplfwlakuhqfaz0j4fs5x2p8y. How ever, where did 0.989 XCH go?

**Reply by loppefaaret** [reply]
you took the 1 XCH out of the wallet, and split it into 0.01 & 0.98... now imagine your wallet has 1000000000 millions of small pockets that can hold coins, just like your wifes bag... instead of putting the 0.98 into the pocket called `xch1rg97f3dvk7wjlsjdcvrtxujv606swa42hpplfwlakuhqfaz0j4fs5x2p8y` - you put the change, 0.98, into another pocket, called something else - it is still in your wallet, just in another pocket inside your wallet

**Reply by loppefaaret** [reply]
block explorers can only track what is inside a pocket - they do not know, and are forever unable to know, what wallet owns which pockets - all they can say is is that the pocket called `xch1rg97f3dvk7wjlsjdcvrtxujv606swa42hpplfwlakuhqfaz0j4fs5x2p8y` send off 1XCH, which split into 0.01 & 0.98, and the 0.1 went to a pocket called `AAAAAAAAAA` and the 0.98 went to a pocket called `BBBBBBBB`... your wallet does not own the `AAAAAAAAA`, that belongs to another wallet, but your wallet _does_ own the pocket called `BBBBBBBB`

**Reply by loppefaaret** [reply]
it is a basic feature of most crypto currencies today, that the wallet generates the next address, whenever it detects it has received a transaction (or more) on one address - this is a form of privacy function that prevents every one else knowing exactly what you have in your wallet in total - all they can see is that 1 address

**Reply by chenus1** [reply]
Oh! Thanks for explanation! Do you also mean that I still own 0.98 xch in my wallet? If future, if I want to check or withdraw 0.98 XCH, do I need to synce the wallet xch1rg97f3dvk7wjlsjdcvrtxujv606swa42hpplfwlakuhqfaz0j4fs5x2p8y or  cold wallet xch16dh6v6kg0t7d36gkkwmcxeqtaf77yagy7w828mugrkh8vjha8ujq8q4fqn ? Thank you for your time!

**Reply by loppefaaret** [reply]
im only _assuming_, i do not know exactly what you did, but i imagine you were sending 0.01 to another address, and the wallet would itself use a new address it owns, for the returning change that was leftover from splitting a 1.00 XCH coin

**Reply by chenus1** [reply]
This is what I did:  
I had 0.01 XCH in the cold wallet---xch16dh6v6kg0t7d36gkkwmcxeqtaf77yagy7w828mugrkh8vjha8ujq8q4fqn
I sent 1 XCH from farming wallet xch1rg97f3dvk7wjlsjdcvrtxujv606swa42hpplfwlakuhqfaz0j4fs5x2p8y to  the cold wallet xch16dh6v6kg0t7d36gkkwmcxeqtaf77yagy7w828mugrkh8vjha8ujq8q4fqn
After I synced the cold wallet and I confirmed that I had 1.01 XCH in the cold wallet after I receive 1 XCH. I then sent 0.01 XCH from the cold wallet to farming wallet. I then synced the farming wallet and deleted the cold wallet key in my farming machine.  I received 0.01 XCH in farming wallet after I synced the farming wallet  several days later. However when I checked the cold wallet again, I noticed that 0.98 XCH is missing in the cold wallet. That I am wondering and very concern about it.  I am concerned that my farming machine is compromised or something is not normal, as I never sent 0.98 XCH to the address xch16n3kejm7t8en5022xe9mq96y7ynfha9g9una2hd85a35jep7accsf9tk6l  
I want to know if the 0.98 XCH is really missing, or it is still in my wallet. I like to avoid this happen in future. Really appreciate!

**Reply by loppefaaret** [reply]
it is now under the address https://xchscan.com/address/xch16n3kejm7t8en5022xe9mq96y7ynfha9g9una2hd85a35jep7accsf9tk6l - the "missing" coins are there - your "not so cold wallet anymore" used a new address in the wallet to split the 1.00 XCH... you can try and sync the "not so cold wallet" again, and you will see it...

**Reply by loppefaaret** [reply]
you can keep using xch16dh6v6kg0t7d36gkkwmcxeqtaf77yagy7w828mugrkh8vjha8ujq8q4fqn - the transactions will lead to the "not so cold wallet"
or you can also use this xch16n3kejm7t8en5022xe9mq96y7ynfha9g9una2hd85a35jep7accsf9tk6l address - these will also land in the "not so cold wallet"

**Reply by chenus1** [reply]
Not sure what it is the difference between "not so cold wallet" and "cold wallet". Do you mean once the "cold wallet" is synced once, it is not "cold wallet" but "no so cold wallet"? Can I create new wallet  with different private key in any other computer, and just change the "farming reward address" and "pool reward address" in "manage reward address" menu of farming machine to the new created wallet address. So if I do not synce the new created wallet, will the wallet be called "cold" and is it will be more safe?

**Reply by loppefaaret** [reply]
thats was exactly my intentions to to make clear - it is not cold hte moment the key was able to have internet access!
yes, you can create a new key, and set farmer reward target address, and have rewards go to it directly

**Reply by chenus1** [reply]
Thank you!

**Comment by BrandtH22** [comment]
@chenus1 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
